src/components/StorageOverview/Inventory/DiskInventoryUtils.js

```diff
@@ -0,0 +1,26 @@
+import { get } from 'lodash';
+
```

Comment:
please rename this file to `diskInventoryUtils`. Only component files start with an uppercase.